planner/core/logical_plan_builder.go

```diff
@@ -3213,6 +3218,23 @@ func (b *PlanBuilder) buildWindowFunctionFrame(ctx context.Context, spec *ast.Wi
 	return frame, err
 }
 
+func (b *PlanBuilder) checkWindowFuncArgs(ctx context.Context, p LogicalPlan, windowFuncExprs []*ast.WindowFuncExpr, windowAggMap map[*ast.AggregateFuncExpr]int) error {
+	for _, windowFuncExpr := range windowFuncExprs {
+		_, _, _, args, err := b.buildProjectionForWindow(ctx, p, &windowFuncExpr.Spec, windowFuncExpr.Args, windowAggMap)
```

Comment:
We can pass an empty spec since it does not matter here and won't do extra work for partition by and order by items.

Reply:
use a dummy &windowSpec{} didn't work for sql :
`select a, b from t order by default(c)`  in case : expression/integration_test.go/TestOrderbyWithDefaultFunc

Reply:
fixed